tensorflow/lite/micro/examples/person_detection/Makefile.inc

```diff
@@ -1,6 +1,3 @@
-$(eval $(call add_third_party_download,$(PERSON_MODEL_URL),$(PERSON_MODEL_MD5),person_model_grayscale,))
-$(eval $(call add_third_party_download,$(RUY_URL),$(RUY_MD5),ruy,))
```

Comment:
removing ruy makes sense, since it is in the primary Makefile. I would siggest leaving person_model_grayscale here to avoid unnecessary downloads for other build targets and variants.

Reply:
If I am not wrong, this file is always included by:

https://github.com/tensorflow/tensorflow/blob/3448ea1f3d321d9d3d0487b1680a75a5c7ce05bd/tensorflow/lite/micro/tools/make/Makefile#L100

This means, person_model_grayscale is added to third_party_download always anyway.